### ISSUE TITLE
[Bug]: Search bar on dark mode has strange background

### ISSUE THREAD

**Issue by emilio** [mozilla-mobile/fenix#25462]
### Steps to reproduce

1. Have phone in dark mode.
2. Have a search widget

### Expected behaviour

The search bar shows correctly 

### Actual behaviour

There's an odd background behind it. Removing the widget and adding it back works around the issue but it comes back after a while.

### Device name

OnePlus 9 

### Android version

Android 12

### Firefox release type

Firefox Nightly

### Firefox version

102.0a1 (Build #2015883435), 5ea4c6533+ AC: 102.0.20220529143103, 89506cd9c7 GV: 102.0a1-20220529090310 AS: 93.1.0

### Additional information

![Screenshot_2022-05-31-11-13-58-39_b783bf344239542886fee7b48fa4b892](https://user-images.githubusercontent.com/1323194/171138830-2f6a0d24-8eb2-4c61-8768-42444343cfdc.jpg)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-20608)

**Comment by Mugurell** [comment]
This would seem to me like a device / launcher issue.
@emilio Does this issue affect other widgets also? Like of Chrome / Google ?

**Comment by emilio** [comment]
@Mugurell I haven't seen this on other widgets (but I don't use anything else than the Firefox search bar  and a clock). I'll try with some others and let you know if I repro with those.

**Comment by delia-pop** [comment]
Hello, @mcarare , @Mugurell , I encountered a similar issue on the latest Nightly from 06/18 with Motorola Moto G30 (Android 11). After adding a widget for Nightly on homescreen, i noticed that it was improperly displayed as a completely black 5x3 block, as in the screenshot above. I assumed it was a device relate issue and tried to reproduce it after stumbling upon this ticket, but with no luck in what regards the incorrect shape. 

However, I successfully reproduced on different devices the following behavior: after a device restart, the Firefox widgets (for Nightly, Beta or RC) are added on homescreen without the logo or the "Search the web" string, which will appear only after repeatedly removing/adding/interacting with the widgets.

STR: 
1. Have Dark theme enabled on device.
2. Perform a device restart.
3. Add a widget to device homescreen for Firefox Nighlty/Beta/RC.
4. Notice that only the mic icon is displayed on the added widget. 

Reproduced on Motorola Moto G30 (Android 11) and Google Pixel 6 (Android 12).

[Widget_MissingLogoAndText.txt](https://github.com/mozilla-mobile/fenix/files/8941871/Widget_MissingLogoAndText.txt)
https://user-images.githubusercontent.com/89388888/174634793-370cc7af-1a48-485d-bb76-4188e9f61fe8.mp4
